links/index.js

```diff
@@ -42,31 +78,72 @@ var page2 = {
         {
             type: 'heading2',
             value: 'Page 2'
+        },
+        {
+            type: 'link',
+            value: 'Back',
+            attrs: {
+                pageId: '1'
+            }
         }
     ]
 };
 
+// Page 3 uses two parameters for an image name and URL.
+var makePage3 = (params) => ({
```

Comment:
Can we pass the parameters explicitly here (image_name and image_url), and leave it to the controller to break them up?